backend/src/main/java/me/quizzl/backend/models/MultipleChoice.java

```diff
@@ -0,0 +1,26 @@
+package me.quizzl.backend.models;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class MultipleChoice extends Question{
```

Comment:
I still don't really know how Hibernate/JPA works yet but I think this might be helpful with regards to what you're looking for: capt.kyn@gmail.com